README.md

```diff
@@ -118,6 +118,58 @@ The backend system would collect all `sensu_check` resources.
   Sensu_check <<||>>
 ```
 
+### Sensu backend cluster
+
+A `sensu-backend` cluster can be defined for fresh installs by defining the necessary `config_hash` values.
+The following examples are using Hiera and assume the `sensu::backend` class is included.
+
+```yaml
+# data/fqdn/sensu-backend1.example.com.yaml
+---
+sensu::backend::config_hash:
+  listen-client-urls: 'http://0.0.0.0:2379'
+  listen-peer-urls: 'http://0.0.0.0:2380'
+  initial-cluster: 'backend1=http://192.168.0.1:2380,backend2=http://192.168.0.2:2380'
+  initial-advertise-peer-urls: "http://%{facts.ipaddress}:2380"
+  initial-cluster-state: 'new'
+  name: 'backend1'
+```
+```yaml
+# data/fqdn/sensu-backend2.example.com.yaml
+---
+sensu::backend::config_hash:
+  listen-client-urls: 'http://0.0.0.0:2379'
+  listen-peer-urls: 'http://0.0.0.0:2380'
+  initial-cluster: 'backend1=http://192.168.0.1:2380,backend2=http://192.168.0.2:2380'
+  initial-advertise-peer-urls: "http://%{facts.ipaddress}:2380"
+  initial-cluster-state: 'new'
+  name: 'backend2'
+```
+
+Adding new members to a cluster requires two steps. First add the member using `sensu_cluster_member` type then second configure and start `sensu-backend` on the new node.
+
+First, add the member:
```

Comment:
Does this mean it takes multiple puppet runs in a certain sequence? If so, could you elaborate?

Reply:
I think so, I'll try to come up with some better wording on order.  It's a really specific order of operations to avoid errors.  Basically add member on existing backend node with sensuctl then start new member with config that differs slightly from existing backend nodes.

Reply:
Updated README, hopefully a bit clearer.  Let me know if needs improvements.